### DISCUSSION TITLE
eth_signTransaction/sign/signTypedV4 support

### DISCUSSION THREAD

**Discussion by leovigna** [wevm/frog-archived#279] [Ideas]
Is it possible to support additional wallet RPC methods? Not very familiar with the frames spec but looking at the eth_sendTransaction implementation it seems that simply changing the method and Paramus would work?

Idk lol. Please let me know if this is even possible.

**Comment by dalechyn** [comment]
Hey, it is currently not supported by fatcaster itself, thus cannot be implemented.

**Comment by dalechyn** [comment]
Supported now by FC and Frog: https://frog.fm/concepts/signatures
